network/load-balancer/reference-content/acls.mdx

```diff
@@ -0,0 +1,116 @@
+
+In this case, we want to reject traffic from source IPs `51.158.99.227` and `151.115.78.39`, and also reject traffic with either of these IP addresses in the `X-Forwarded-For` HTTP header. All other traffic should be allowed access.
+
+Construct two ACLs:
+- **ACL Priority 1**: `Deny` - `Enable condition` - `IF MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic from the two specified IPs. All traffic not from these IPs will pass to the next ACL.
+- **ACL Priority 2**: `Deny` - `Enable condition` - `IF MATCH` - `HTTP filter - Header` - `Header name: X-Forwarded-For` `Filters`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic that has the specified IP values in the specified header. All traffic without these values in that header, which hasn't already been rejected by the previous ACL, will by default pass to the backend since there are no further ACLs in the list.
+
+<Message type="note">
+
+  In this example, you could switch the priority order of the ACLs without any effect.
+
+</Message>
+
+### Reject traffic from certain IPs, and with HTTP header values, unless it has a specific HTTP request path
+
+In this case, we want to achieve the same as with our previous example, **however**, if that same traffic is requesting an HTTP request path that begins with `/catalog` we want to allow access nonetheless.
+
+Construct three ACLs:
+- **ACL Priority 1**: `Allow` - `Enable condition` - `IF MATCH` - `HTTP filter` `Path begins with`: `/catalog`. This ACL will allow any traffic with this at the beginning of its HTTP header to pass directly to the backend. All traffic without this header path will pass to the next ACL.
+- **ACL Priority 2**: `Deny` - `Enable condition` - `IF MATCH` - `IP filter`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic from the two specified IPs. All traffic not from these IPs will pass to the next ACL.
+- **ACL Priority 3**: `Deny` - `Enable condition` - `IF MATCH` - `HTTP filter - Header` - `Header name: X-Forwarded-For` `Filters`: `51.158.99.227`, `151.115.78.39`. This ACL will reject traffic that has the specified IP values in the specified header. All traffic without these values in that header, which has not already been allowed by the first ACL or rejected by the second ACL, will by default pass to the backend since there are no further ACLs in the list.
+
+## Using ACLs to redirect traffic
+
+Use an ACL with the **redirect** action to redirect traffic. You can choose to redirect traffic using the original URI but with a different schema (HTTP or HTTPs), or define a new URI to redirect to. You can also choose whether to make the ACL conditional or unconditional, just as with our previous examples.
+
+### Redirecting from HTTP to HTTPS
+
+In this case, we want to redirect all incoming HTTP requests to HTTPS. Our Load Balancer already has a frontend configured for HTTPS requests (via an [SSL/TLS certificate](/network/load-balancer/how-to/add-certificate).
```

Comment:
Maybe also worth mentioning the frontend must be linked to an HTTP backend (to make sure it's an HTTP frontend) because frontend type is implied by the backend type ...